pyproject.toml

```diff
@@ -11,9 +11,9 @@ packages = [
   
 
 [tool.poetry.dependencies]
-python = "^3.6.2,<3.10"
+python = "^3.10"
```

Comment:
We cannot do that as it would end support for python <3.10. 
Rather try it with `^3.6.2,<=3.10`.